packages/react/src/components/PageList/PageList.module.css

```
@@ -0,0 +1,55 @@
+  flex: 1;
+  min-width: var(--size-spacing-2xl);
+  border-bottom: var(--size-border-m) solid var(--color-semantic-neutral-300);
+  margin-inline: var(--size-scale-8);
+  margin-block-start: calc((var(--typography-line-height-m-normal) - var(--size-border-m)) / 2);
+}
+
+.date {
+  flex-shrink: 0;
+  white-space: nowrap;
+  margin-block-start: calc(
+    (var(--typography-line-height-m-dense) - var(--typography-line-height-s-dense)) /
```

Stylelint (17.11.1)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)